pilot/docker/Dockerfile.proxyv2

```diff
@@ -45,6 +48,7 @@ ENV ISTIO_META_ISTIO_PROXY_SHA $proxy_version
 ENV ISTIO_META_ISTIO_VERSION $istio_version
 
 COPY pilot-agent /usr/local/bin/pilot-agent
+RUN setcap cap_net_admin,cap_net_raw=+eip /usr/local/bin/pilot-agent
```

Comment:
Is it possible to run `setcap` on the binary before copying it into docker or will that be stripped on copy?

Reply:
Can be run before (it worked with iptables)

Reply:
Or at least it worked from Docker to Docker